server/server.go

```diff
@@ -9,6 +9,28 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 )
 
+const usage = `<html><head><title>Not found</title></head><body>
+<h1>Not found</h1>
+
+<p>
+Give a username and get an avatar in return: <a href="https://avatars.charlyx.dev/twitter?username=charlyx">https://avatars.charlyx.dev/twitter?username=charlyx</a>
+</p>
+
+<p>
+You can ask for variant sizings such as "bigger", "mini" and "original" (default size being "normal").
+</p>
+
+<p>
+Example: <a href="https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger">https://avatars.charlyx.dev/twitter?username=charlyx&size=bigger</a>
+</p>
+</body>
+</html>`
+
+func defaultHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusNotFound)
+	fmt.Fprintf(w, usage)
```

Comment:
`http.Error(w, usage, http.StatusNotFound)` might do the exact same thing using one line.

Reply:
`http.Error` wraps the content with a full html page and `<pre></pre>` markup
I'll go with `w.Write()`